src/app/components/authenticated-v2/subpages/help-page/help-page.component.html

```diff
@@ -1,137 +1,166 @@
-      I keep updating and improving this tool, but beware that it takes a bit, as I work on it on my spare time.
-      Feel free to follow me on Twitter <a href="https://twitter.com/MijagoCoding/" target="_blank"><img
-      alt="Me on twitter" class="logo-twitter" src="assets/logo-twitter.png"></a>, I occasionally post updates there.
-      If you like my tools and want to support me, head to
-      <a href="https://ko-fi.com/mijago" target="_blank">ko-fi.com</a> and buy me a coffee! ❤
-    </mat-card-content>
-  </mat-card>
+    <mat-card>
+        <mat-card-title> How do I use D2ArmorPicker? </mat-card-title>
+        <mat-card-content>
+            Follow these few steps:
+            <ol>
+                <li>Login. You already did this!</li>
+                <li>
+                    [Optional] Pick an exotic you want to use. The tool will only display results
+                    for this exotic. You can always undo this.
+                </li>
+                <li>
+                    [Optional] Select additional mods/fragments that give a bonus (or penalty).
+                    These are important if you want certain stat combinations even if you have
+                    penalties on your gear.
+                </li>
+                <li>
+                    Select the stat distribution you seek. This is the heart of D2ArmorPicker. You
+                    just click 100 recovery and it will display you builds that have 100 recovery.
+                    <i>Note that it automatically adds stat mods if necessary.</i>
+                </li>
+                <li>
+                    Now look to the result table. On a normal desktop it should be on the right, on
+                    smaller devices it might move below the configuration. The table lists multiple
+                    results and you can sort it by your requirements.<br />
+                    <b>Click</b> on an entry to open look the detailed view:
+                    <div class="preview-image-container">
+                        <img
+                            alt="Preview of the results table"
+                            class="preview-image"
+                            src="assets/armor-images/preview_result_details.png" />
+                    </div>
+                    The example above shows a build that utilizes Dunemarchers and a
+                    zero-waste-build. To achieve it, I have to move all the items in my inventory
+                    and equip them. After this, I <i>masterwork</i> them and <i>add the mods</i> as
+                    the table displays them:
+                    <ul>
+                        <li><b>Discipline:</b> One major mod, one minor mod;</li>
+                        <li><b>Intellect:</b> One major mod, one minor mod;</li>
+                        <li><b>Strength:</b> One major mod;</li>
+                        <li
+                            >I also add Powerful Friends and Radiant Light, as I chose them in the
```

Comment:
This is outdated

Reply:
I would make a separate ticket for the  Help Page.